tests/test_views.py

```diff
@@ -164,3 +165,31 @@ def test_chat_response(self, mock_mode, mock_enrollment, mock_role, mock_waffle,
 
         render_args = mock_render.call_args.args
         self.assertIn(test_unit_id, render_args)
+
+
+@ddt.ddt
+class LearningAssistantEnabledViewTests(LoggedInTestCase):
+    """
+    Test for the LearningAssistantEnabledView
+    """
+    sys.modules['lms.djangoapps.courseware.access'] = MagicMock()
+    sys.modules['lms.djangoapps.courseware.toggles'] = MagicMock()
+    sys.modules['common.djangoapps.course_modes.models'] = MagicMock()
+    sys.modules['common.djangoapps.student.models'] = MagicMock()
+
+    def setUp(self):
+        super().setUp()
+        self.course_id = 'course-v1:edx+test+23'
+
+    @ddt.data(
+        (True, True),
```

Comment:
[nit] since these are the same (in other words, if the mock value is true, then the expected should also be true) would you need to have both? Could you reduce to one so that there are fewer params?

Reply:
I could see the value of having both for readability though, so would defer to you!

Reply:
It feels a little more explicit to me, personally. This way, I'm saying "this is the input, and this is the expected output". In my mind, if I set the ddt data to be `(True, False)` (i.e. "this is the input"), it requires reading the test in order to understand what the expected case is. That's not a huge deal, but I prefer to be explicit, when I can.

Reply:
I think that's a good decision to be more explicit!